src/transport/PairingSession.h

```diff
@@ -36,6 +36,28 @@ class DLL_EXPORT PairingSession
     PairingSession() {}
     virtual ~PairingSession() {}
 
+    NodeId GetPeerNodeId() const { return mPeerNodeId; }
+    void SetPeerNodeId(NodeId peerNodeId) { mPeerNodeId = peerNodeId; }
+
+    uint16_t GetPeerKeyId() const { return mPeerKeyId; }
+    void SetPeerKeyId(uint16_t id) { mPeerKeyId = id; }
+    bool IsValidPeerKeyId() const { return mPeerKeyId != kInvalidKeyId; }
+
+    uint16_t GetLocalKeyId() const { return mLocalKeyId; }
```

Comment:
Need to document semantics.  I'd expect this to return a valid value if the session object is in a valid state at all...

Reply:
Added a TODO, will be initialized on construction